test.py

```diff
@@ -7,48 +7,53 @@
 
 import pytest
 
-from ordered_set import OrderedSet
+from ordered_set import FrozenOrderedSet, OrderedSet
 
 
-def test_pickle():
-    set1 = OrderedSet('abracadabra')
+@pytest.mark.parametrize("cls", [OrderedSet, FrozenOrderedSet])
+def test_pickle(cls):
```

Comment:
This is neat. Pytest will run the test twice, once with `OrderedSet` and another with `FrozenOrderedSet`.